Protacon.RxMq.AzureServiceBus/Topic/AzureTopicPublisher.cs

```diff
@@ -38,7 +38,13 @@ private class Binding : IDisposable
                 _logger = logger;
                 _settings = settings;
                 queueManagement.CreateTopicIfMissing(topic, type);
-                _topicClient = new TopicClient(settings.ConnectionString, topic);
+                
+                var retryPolicy = new RetryExponential(
+                    TimeSpan.FromSeconds(5),
+                    TimeSpan.FromSeconds(30),
+                    3
+                );
+                _topicClient = new TopicClient(settings.ConnectionString, topic, retryPolicy);
```

Comment:
I would prefer if this setting would be fetched from AzureTopicMqSettings.cs (or perhaps in the base class) so it would be configurable instead of hardcoded.

Also, if this is hardcoded as a default, are there some events when this is not a good default?

Reply:
I agree on this. The values should be set in the Topic settings instead.